templates/production.ts

```diff
@@ -0,0 +1,368 @@
+    }
+
+    h2 {
+      margin-bottom: 15px;
+      padding: 0;
+      font-size: 16px;
+      line-height: 25px;
+      font-weight: bold;
+      color: #323A45;
+      display: inline;
+    }
+
+    p {
+      margin-bottom: 20px;
+      font-size: 16px;
+      line-height: 25px;
+      color: #323A45;
+    }
+  </style>
+</head>
+<div class="card">
+  <div class="header"><img src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/VA-Lighhouse-APIs-email.png" alt="logo"/></div>
+  <p>We’ve received your request for production access. It’s good to remember that getting production access requires
+    multiple steps and can take less than a week to over a month, depending on the API. For open data APIs,
+    production access takes a week or less and no demo is required.</p>
+  <h1><strong>What’s next?</strong></h1>
+  <div class="steps-container">
+    <img class="steps" src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/Done.png" alt="Step one, done"/><h2>Start developing.</h2>
+  </div>
+  <div class="steps-container">
+    <img class="steps" src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/Done.png" alt="Step two, done"/>
+    <h2>Submit your production access request.</h2>
+  </div>
+  <p class="sub">
+    We’ll review your information and notify you within 1-2 weeks if there are any changes we need.
+  </p>
+  <div class="steps-container">
+    <img class="steps" src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/step-3-selected.png" alt="Step three, todo"/>
+    <h2>
+      <a href="https://developer.va.gov/onboarding/prepare-for-and-complete-a-demo">
+        Make any needed technical or policy changes, then complete a demo.
+      </a>
+    </h2>
+  </div>
+  <p class="sub">The faster you complete any changes, the faster we can schedule a demo. Once your changes
+    are made, we aim to
+    schedule the demo within a week. Demos generally last from 30 to 60 minutes. </p>
+  <div class="steps-container">
+    <img class="steps" src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/step-4-disabled.png" alt="Step four"/>
+    <h2>Receive production access.</h2>
+  </div>
+  <p class="sub">
+    After the demo is complete and any final items are wrapped up, production access is granted within a week.
+  </p>
+  <p>
+    Interested in getting access to another API? Read the API docs or request a new API key.
+  </p>
+  <p>
+    <a href="https://developer.va.gov/onboarding/prepare-for-and-complete-a-demo">
+      <u>Preparing for a demo</u>
+    </a> |
+    <a href="https://developer.va.gov/onboarding/working-with-lighthouse-apis">
+      <u>Working with Lighthouse APIs</u>
+    </a> |
+    <a href="https://developer.va.gov/support/contact-us">
+      <u>Contact us</u>
+    </a>
+  </p>
+</div>
+`;
+export const PRODUCTION_ACCESS_SUPPORT_TEMPLATE = `<h2>Basic Information</h2>
+<ul>
```

Comment:
personally don't think the `ul` format is super readable for this sort of lengthy information (i know we use it for other support emails), but as long as support can read it, i'm fine with it

Reply:
I'd leave that decision up to support to decide later on. The format can always be updated.